src/utils/helpers/shuffleArray.ts

```diff
@@ -0,0 +1,9 @@
+// https://en.wikipedia.org/wiki/Fisher-Yates_shuffle
+export function shuffleArray(array: unknown[]): void {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    const temp = array[i];
```

Comment:
If you hate the temp as much as I do you can
```suggestion
    [array[i], array[j]] = [array[j], array[i]];
```

Reply:
True, I was just refactoring it as-is from where it was. Probably both compile to the same thing.